### DISCUSSION TITLE
Hardware requirements

### DISCUSSION THREAD

**Discussion by Kaloszer** [intelowlproject/IntelOwl#2479] [General]
I couldn't find hardware requirements for this project - it would be nice to know what is the minimum 'working' hardware reqs (eg. RAM/Disk size/vCPU - or equivalent VM size in `Azure` and `AWS`). It would also be nice to have a benchmark telling what the time is to set up an env (initialize + start prod up).

**Comment by mlodic** [comment]
you are right, thanks for suggesting.

There are several types of deployment so the answers is long, I'll add something in the doc for the upcoming version.

Meanwhile, for a basic test, a 2CPU 4GB RAM 20GB Disk is enough.

**Reply by Kaloszer** [reply]
@mlodic Thanks, here goes my 4$/month VM - gotta be less cheap... sometimes 😭

**Reply by Kaloszer** [reply]
For Azure lowest viable VM size is:
`Standard B2s`

`Standard B1ms` initializes w/o --all_analyzers 
`Standard B1ms` won't initialize with --all_analyzers 
Probably hits out of memory as the cli hangs up and the VM is unresponsive.
